bodhi/tests/client/test___init__.py

```diff
@@ -1602,6 +1602,41 @@ def test_edit_with_errors(self, send_request):
         self.assertEqual(result.exit_code, 1)
         self.assertEqual(result.output, ("ERROR: an error was encountered... :(\n"))
 
+    @mock.patch('bodhi.client.bindings.BodhiClient.csrf',
+                mock.MagicMock(return_value='a_csrf_token'))
+    @mock.patch('bodhi.client.bindings.BodhiClient.send_request',
+                return_value=client_test_data.EXAMPLE_RELEASE_MUNCH, autospec=True)
+    def test_edit_mail_template(self, send_request):
+        """
+        Assert correct behavior while editing 'mail_template' name.
+        """
+        runner = testing.CliRunner()
+
+        result = runner.invoke(
+            client.edit_release,
+            ['--name', 'F27', '--mail-template', 'edited_fedora_errata_template', '--url',
+             'http://localhost:6543', '--username', 'bowlofeggs', '--password', 's3kr3t'])
```

Comment:
We don't need the url, username, or password arguments here and the test will be cleaner without them.